README.md

```diff
@@ -1,16 +1,17 @@
-# GitHub Universe Workshop Day 2017
-## GitHub and the Internet of Things: Automate IoT Hardware
-Arduino and IoT information and code for GitHub Universe Workshop Day 2017
+# IntoTech 2017
+## Arduino Workshop
+Let’s break the ice and get started in the makers’ movement by assembling a pre-designed hardware project from scratch! Come join as as we hit the ground running by prototyping your very own electronics project using Arduino and C++. No electronics or programming knowledge is required! 
+
+Requirements: Please bring along a laptop. You don’t need to have any software preinstalled, however please download the [Arduino installer](https://www.arduino.cc/en/Main/Software) ahead of the workshop. We recommend listening to the The IoT Talk from Jamie happening directly before this workshop, as they are connected! 
+
+This workshop is supported with material by [Conrad Electronics](https://www.conrad.at/)
 
 ### Prerequisites
-Before arriving to the workshop, please make sure you have the following items installed on your laptop:
+These items are used for this workshop:
 - [Arduino IDE](https://www.arduino.cc/en/Main/Software)
-- [PlatformIO](http://docs.platformio.org/en/latest/installation.html)
-  - [Python 2.7](https://www.python.org/downloads/) - note that Python 3 will not work with PlatformIO
-  - [Atom](https://atom.io/) with [PlatformIO package](http://platformio.org/get-started/ide?install) and [Clang](http://clang.llvm.org/) if required (Windows users)
-- [Travis](https://docs.travis-ci.com/user/getting-started/) familiarity, especially for [C++](https://docs.travis-ci.com/user/languages/cpp/)
 - [Liquid Crystal Library](https://www.arduino.cc/en/Reference/LiquidCrystal)
 - [DHT Library](https://github.com/adafruit/DHT-sensor-library)
```

Comment:
We should probably leave out the URL to this library so that users don't try installing the library from the GitHub page.  It'd also be useful to leave a quick one-liner linking to [this document](https://www.arduino.cc/en/Guide/Libraries#toc2) on how to install the library via the IDE.

Reply:
Removed link and added note that it comes with the IDE. Added link to library info.